src/ru/skypro/Main.java

```diff
@@ -1,8 +1,63 @@
 package ru.skypro;
 
+import java.sql.Array;
+import java.util.*;
+
 public class Main {
 
     public static void main(String[] args) {
-	// write your code here
+
+        //getOddNumbers();
+        //getEvenNumbers();
+        //getUniqWords();
+        getUniqWordsCount();
+
+
+    }
+    public static void getOddNumbers () {
+        List<Integer> nums = new ArrayList<>(Arrays.asList(1, 1, 2, 3, 4, 4, 5, 5, 6, 7));
+        List<Integer> numsOdd = new ArrayList<>();
+        for (int i = 0; i < nums.size(); i++) {
+            int currentNumber = nums.get(i);
+            if (currentNumber % 2 != 0) {
+                numsOdd.add(currentNumber);
+            }
+        }
+        System.out.println(numsOdd);
+    }
+
+    public static void getEvenNumbers() {
+        List<Integer> nums2 = new ArrayList<>(Arrays.asList(1, 1, 2, 3, 4, 4, 5, 5, 6, 7));
+        List<Integer> numsEven = new ArrayList<>();
```

Comment:
можно сразу создать лист в котором будут храниться уникальные значения
```suggestion
        List<Integer> numsEven = new ArrayList<>(new HashSet<>(nums))
```

Reply:
Нужно добавить сортировку, можно использовать Collections.sort() или создать TreeSet